### PR TITLE
added environment variable documentation

### PR DESCRIPTION
**Pull request by seunbayo**

### CODE REVIEW

File: versioned_docs/version-3.0.0-beta/env-var.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+id: env-var
+title: Environment variable
+sidebar_label: Environment variable
```
Comment (rekpero) [inline versioned_docs/version-3.0.0-beta/env-var.md:4]: There is already a page for Env var inside the project category. We don't need this page here. Please transfer all the content to that page

File: versioned_docs/version-3.0.0-beta/env-var.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+slug: /env-var
+---
+
+## Domain got Environment
```
Comment (rekpero) [inline versioned_docs/version-3.0.0-beta/env-var.md:8]: I think you should mention this on the domain page, not here. There is no context with the environment variable in this section.

Comment (rekpero) [reply versioned_docs/version-3.0.0-beta/env-var.md:8]: Also this is only applicable for centralised domains and subdomains

File: versioned_docs/version-3.0.0-beta/env-var.md (hunk 1 above)
Comment (rekpero) [reply versioned_docs/version-3.0.0-beta/env-var.md:4]: https://docs.spheron.network/projects/environment-variables

**Review by rekpero — CHANGES_REQUESTED** [review]
@seunbayo please check my comments

File: versioned_docs/version-3.0.0-beta/projects/domain-and-https/centralized-domain/overview-custom-domain.md [hunk 3]
```diff
@@ -58,3 +58,15 @@ Earlier, when users attached their domain to a project, they had to add some rec
...
+
+## Domain got Environment
+
+Spheron users can attach a deployment environment with a particular domain. Attaching a deployment environment to the domain will give users the flexibility to segregate their domains based on the environment of a project. That means, that if the deployment of a deployment environment is finished, the domain attached to the deployment environment will get updated with the new deployment link.
+
+<img src={require('@site/static/img/3.0.0/dge.png').default} />
+
+:::tip Note
+
+This is only applicable to centralized and subdomains
```
Comment (rekpero) [inline versioned_docs/version-3.0.0-beta/projects/domain-and-https/centralized-domain/overview-custom-domain.md:70]:
```suggestion
This is only applicable to centralized domains and subdomains
```

File: versioned_docs/version-3.0.0-beta/projects/environment-variables.md [hunk 4]
```diff
@@ -25,14 +25,52 @@ Adding Environment Variables during new deployment won't work if project already
 
 :::
 
-<font size="4"><b>2. In Project Settings</b></font>
+## Variable got Environment
 
-You can add an environment variable by going to **Project Settings -> Environment Variables** as well.
+<strong>you can add an environment variable by going to Project Settings -> Environment Variables </strong>
```
Comment (rekpero) [inline versioned_docs/version-3.0.0-beta/projects/environment-variables.md:30]:
```suggestion
<strong>You can add an environment variable by going to Project Settings -> Environment Variables </strong>
```

**Review by rekpero — CHANGES_REQUESTED** [review]
@seunbayo some small fix

**Comment by seunbayo** [comment]
@rekpero  fixed

**Review by rekpero — APPROVED** [review]

**Pull request merged**